docs/operation/volume_management/volume_creation_deletion_gui.rst

```diff
@@ -1,68 +1,70 @@
 Volume Management using the UI
 ==============================
 
-This section describes the creation and deletion of MetalK8s **Volume**
-using the MetalK8s UI.
-In order to create Volumes you need to have StorageClass objects registered in
-your cluster. See :doc:`/operation/volume_management/storageclass_creation`
+This section describes how to create and delete a MetalK8s **Volume**
+using the MetalK8s **UI**.
 
-Volume Creation
----------------
+.. important::
+
+   **StorageClass** objects must be registered in your cluster to create
+   volumes. For more information refer to :doc:`/operation/volume_management/storageclass_creation`.
+
+Creating a Volume
+-----------------
 
-To access the UI, refer to
-:ref:`this procedure <installation-services-admin-ui>`
+.. note::
 
-#. Navigate to the **Nodes** list page, by clicking the button in the sidebar:
+   To access the MetalK8s UI, refer to
+   :ref:`this procedure <installation-services-admin-ui>`.
+
+#. Click **Nodes** on the sidebar to access the Node list.
```

Comment:
We would need to update the screenshots are they are not completely different. The workflow remains valid though ...
You can use the following URL to make screenshots: https://10.200.6.104:8443/nodes/jhkj/volumes